appsec-kit-backend/src/main/java/com/vaadin/appsec/backend/AppSecScanEvent.java

```diff
@@ -23,4 +28,22 @@ public class AppSecScanEvent extends EventObject {
     public AppSecService getSource() {
         return (AppSecService) super.getSource();
     }
+
+    /**
+     * Gets the list of new vulnerabilities found on this scan. A vulnerability
+     * is considered new if there is not a developer assessment data for that
+     * vulnerability.
```

Comment:
Based on this description a vulnerability is new if there is no developer interaction yet to that vulnerability. Should we also check the vulnerability status if that is `Not set`? `Not set` is the default and the vulnerability can be saved to the list with this status without any developer assessment or analysis. Checking only the identifier doesn't take in account those.

Reply:
Yes, that is true. I was just thinking if we want to handle those vulnerabilities as still new and also as a reminder for the developer. But yes, there will be at least once interacted with the vulnerability.

Reply:
Good point, we can further improve with a more solid approach e.g. adding new methods to the event like `getUnhandledVulnerabilities` or `getExploitableVulnerabilitites`.